### PR TITLE
OrderingAPI

### PR DESCRIPTION
**Pull request by jkeuleya**
OrderingAPI implementation

### CODE REVIEW

File: lib/amazon_business_api/client.rb [hunk 1]
```diff
@@ -79,10 +79,17 @@ def request_access_token # rubocop:disable Metrics/MethodLength
       data
     end
 
+    def headers
+      {
+        'x-amz-access-token' => access_token,
+        'x-amz-user-email' => ENV['AMAZON_BUSINESS_API_EMAIL']
```
Comment (jkeuleya) [inline lib/amazon_business_api/client.rb]: We don't need line 85 everytime. I had to add it for OrderingAPI calls. Quick patch.

File: lib/amazon_business_api/resources/place_order_expectations/amount.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```
Comment (jkeuleya) [inline lib/amazon_business_api/resources/place_order_expectations/amount.rb:1]: DRY with money?

File: lib/amazon_business_api/resources/money.rb [hunk 3]
```diff
@@ -4,6 +4,6 @@ module AmazonBusinessApi
   class Money < AmazonBusinessApi::Resource
     # https://developer-docs.amazon.com/amazon-business/docs/reporting-api-v1-reference-1
     attribute :currency_code, type: LedgerSync::Type::String
-    attribute :amount, type: LedgerSync::Type::String
+    attribute :amount, type: LedgerSync::Type::String # TODO: Float?
```
Comment (jkeuleya) [inline lib/amazon_business_api/resources/money.rb:7]: Why not a float?

File: lib/amazon_business_api/client.rb (hunk 1 above)
Comment (KostasKostogloy) [reply lib/amazon_business_api/client.rb]: we have `email` available on our client

File: lib/amazon_business_api/place_order/operations/create.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+        private
+
+        def ledger_resource_type_for_path
+          "/ordering/2022-10-30/orders"
+        end
+
+        def url
+          ledger_resource_type_for_path
+        end
```
Comment (KostasKostogloy) [inline lib/amazon_business_api/place_order/operations/create.rb:11]: let's just have the url return it directly

File: lib/amazon_business_api/place_order/operations/find.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+          params do
+            required(:external_id).filled(:string)
+          end
+        end
+
+        private
+
+        def hash_to_deserialize
+          response.body
+        end
```
Comment (KostasKostogloy) [inline lib/amazon_business_api/place_order/operations/find.rb]: this is redundant, it's defined the same on the base operation

**Pull request merged**